lib/aur-sync

```diff
@@ -357,7 +357,7 @@ if ((build)); then
     build_args+=(-d "$repo" -r "$root" -a "$tmp"/queue)
 
     # check if dependency graph is valid
-    printf '%s\0' "$tmp_view"/*/.SRCINFO | xargs -0 aur graph >/dev/null
+    aur graph "$tmp_view"/*/.SRCINFO >/dev/null
```

Comment:
I went with a simple glob here to avoid an additional parallel invocation... I guess we're pretty unlikely to reach ARG_MAX here anyway.